var/spack/repos/builtin/packages/lodepng/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```

Comment:
lodepng and sjpeg are deps of libjxl, but there doesn't seem to be a way to use external installations of these. So these packages don't necessarily have to be in this PR, let me know if you want me to move them to a separate PR to make this PR simpler to review.